nebari/constants.py

```diff
@@ -5,8 +5,8 @@
 
 HIGHEST_SUPPORTED_K8S_VERSION = "1.23.5"
 
-DEFAULT_QHUB_DASK_VERSION = "0.4.5"
-DEFAULT_QHUB_IMAGE_TAG = "main"
+DEFAULT_NEBARI_DASK_VERSION = "0.4.5"
```

Comment:
This needs to be fixed if we are going to make the first release of nebari-dask as `2022.10.1`

Reply:
I'm not seeing this... I see `NEBARI`

Reply:
<img width="1460" alt="Screen Shot 2022-10-26 at 14 07 02" src="https://user-images.githubusercontent.com/42120229/198010863-73028042-f578-43ab-91ee-c7fa7a699695.png">

Reply:
Nvm, I see you want the tag updated. Done.

Reply:
she means the version itself, wich needs to be changed from `0.4.5 -> 2022.10.1
`